girder/models/folder.py

```diff
@@ -309,7 +309,6 @@ def remove(self, folder, progress=None, **kwargs):
             if progress:
                 progress.update(increment=1, message='Deleted item ' +
                                 item['name'])
-        items.close()
```

Comment:
In cpython it will be closed via GC after going out of scope, so you'd have to put each in its own function.